src/asciidoc/docs/_includes/misc.adoc

```diff
@@ -0,0 +1,17 @@
+= Miscellaneous
```

Comment:
What do you think of naming this file more descriptively ("Node versions" or something like that)? It seems like "Miscellaneous" would make it harder to find and understand. Then, we can just use the "all" tag and not avoid some complication that doesn't provide additional value at the moment.

In general, something to put on our list of standards/practices is how and when to use includes so that all authors can understand, use, and maintain them.

Reply:
I'm thinking more random snippets won't have a logical home. So this file would be for those snippets, instead of creating one include file for each.